commands/cloudapi-v6/query/query_params.go

```diff
@@ -112,15 +144,15 @@ func getFilters(args []string, cmd *core.Command) (map[string][]string, error) {
 
 // isValidFilter will return true if the filter is part
 // of the available filters array and false if is not.
-func isValidFilter(filter string, availableFiltersObjs ...[]string) bool {
+func isValidFilter(filter string, availableFiltersObjs ...[]string) (string, error) {
 	for _, availableFilters := range availableFiltersObjs {
 		for _, availableFilter := range availableFilters {
 			if strings.ToLower(availableFilter) == strings.ToLower(filter) {
-				return true
+				return availableFilter, nil
 			}
 		}
 	}
-	return false
+	return "", fmt.Errorf("TODO")
```

Comment:
todo?

Reply:
thanks, renamed to `getFirstValidFilter` and the error is now

```
fmt.Errorf("%s is not case-insensitively equal to any of %s",
			filter, strings.Join(availableFilters, ", "))
```